_post/2022-09-22-hyotester-reflection.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+author: tommytester
+title: "Tommy's Reflection on the Class So Far"
+---
+
+Here's the program I'm embedding:
+
+    <iframe src="https://trinket.io/embed/python/c26c35e489" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+    
+    
+    ```
```

Comment:
The code block should work properly without the spaces before  ```.

Reply:
Good observation! It's better style to put the backticks on the beginning of the line, but I'm not sure whether they'll display on our website like this or not. On Github, they do:
~~~
   ```
Is this formatted like code?
```
~~~
produces:
   ```
Is this formatted like code?
```

How something gets rendered depends on the markdown parser used, a little bit like whether a specific browser can render something.

Reply:
So, in this case I think it'd be a fun experiment to merge this code and see what it looks like.